main.go

```diff
@@ -21,8 +21,9 @@ import (
 func main() {
 	inPath := flag.String("in", "-", "input file, url, or - for stdin")
 	outPath := flag.String("out", "-", "output file, or - for stdout")
+	method := flag.String("method", "nearest", "image resize method - 'nearest', 'bicubic', 'bilinear', 'mitchell', 'lanczos2', or 'lanzoc3'")
```

Comment:
You should probably validate that the `method` value you've received is valid: that it's contained in the keys of the `methods` maps